src/gui/mzroll/mainwindow.cpp

```diff
@@ -320,18 +319,24 @@ using namespace mzUtils;
 	galleryWidget = new GalleryWidget(this);
 	bookmarkedPeaks = new BookmarkTableDockWidget(this);
 
-	alignmentPolyVizDockWidget = new AlignmentPolyVizDockWidget(this);
-	alignmentPolyVizDockWidget->setWidget(alignmentPolyVizPlot);
+    sampleRtWidget = new SampleRtWidget(this);
+    sampleRtWidget->setWidget(sampleRtVizPlot);
 
 	isotopePlotDockWidget = new IsotopePlotDockWidget(this);
 	isotopePlotDockWidget->setWidget(customPlot);
 
 	//treemap	 = 	  new TreeMap(this);
 	//peaksPanel	= new TreeDockWidget(this,"Group Information", 1);
 	spectraDockWidget = createDockWidget("Spectra", spectraWidget);
-	alignmentVizDockWidget = createDockWidget("AlignmentVisualization", alignmentVizPlot);
-	alignmentVizAllGroupsDockWidget = createDockWidget("AlignmentVisualizationForAllGroups", alignmentVizAllGroupsPlot);
-	pathwayDockWidget = createDockWidget("PathwayViewer", pathwayWidget);
+
+    groupRtDockWidget = createDockWidget("Per Group Alignment Visualization", groupRtVizPlot); 
```

Comment:
Please change the widget names (on the UI) to "Per Group Deviation", "All Groups Deviation" and "Sample Deviation"

Reply:
Done